root/user/UserProfile.js

```diff
@@ -123,7 +123,7 @@ const UserProfileInformation = withCatalystContext(({
     $c.user && !$c.user.has_confirmed_email_address) ? (
       <strong>
         {l(`Your homepage and biography will not show
-            until you have confirmed your email.`)}
+            until you have verified your email.`)}
```

Comment:
Other occurrences are fine, and I may be wrong, but I think in this specific case the wording was correct:

* MusicBrainz _verify_ your email address (by sending an email to you).
* You _confirm_ your email address to MusicBrainz (by clicking the link in the email).

That is, you don’t _verify_ your own email address, you confirm it to MusicBrainz.

Reply:
I suspect it's confusing to have both words. If you want to be more precise, what about "until you have completed the email verification process"?

Reply:
Since I can't edit this PR, merged and will add a commit changing that string.